### PR TITLE
Added and updates some readmes

### PR DESCRIPTION
**Pull request by bettinaheim**
... such that I can directly link to the examples folder in the org readme. 
I fixed all formatting issues to comply with formatting rules, and the link check should also pass once I made the spec repo public.

### CODE REVIEW

File: examples/jit/README.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
+# Examples: pyqir_jit
+
+The 'pyqir_jit' package provides an easy way to execute generated QIR for the
```
Comment (idavis) [inline examples/jit/README.md:3]: `pyqir_jit`

**Review by idavis — APPROVED** [review]
@bettinaheim Gate it a look through, only saw the one minor issue.

File: examples/jit/README.md (hunk 1 above)
Comment (bettinaheim) [reply examples/jit/README.md:3]: fixed here and where I copied it from

**Comment by bettinaheim** [comment]
> I fixed all formatting issues to comply with formatting rules, and the link check should also pass once I made the spec repo public.

Thanks, @idavis! I fixed all formatting issues to comply with formatting rules, and the link check should also pass once I made the spec repo public.

**Pull request merged**